CoreScriptsRoot/Modules/PlayerlistModule.lua

```diff
@@ -781,7 +782,27 @@ function popupHidden()
 end
 playerDropDown.HiddenSignal:connect(popupHidden)
 
+local function openPlatformProfileUI(rbxUid)
+	if not rbxUid or rbxUid < 1 then return end
+	pcall(function()
+		local platformService = game:GetService('PlatformService')
+		if platformService then
+			local platformId = platformService:GetPlatformId(rbxUid)
+			if platformId and #platformId > 0 then
+				platformService:PopupProfileUI(Enum.UserInputType.Gamepad1, platformId)
+			end
+		end
+	end)
+end
+
 local function onEntryFrameSelected(selectedFrame, selectedPlayer)
+	if isTenFootInterface then
```

Comment:
What I was trying to get at was that we check `isTenFootInterface` and return early, but the function it calls may do nothing depending on whether the platform supports this behavior. Is this the desired behavior?